web/src/components/UserLookupCell/UserLookupCell.js

```diff
@@ -7,6 +7,9 @@ export const QUERY = gql`
     users {
       id
       name
+      UserRole {
```

Comment:
this allows us access to the user's refenced roles.  the idea is maybe we forgo groups and instead use "roles".
Imagine;
I'll assign this to whoever has access to the assets!
Look there's Adam Anderson (AssetWorker)